src/svg.rs

```diff
@@ -132,7 +158,7 @@ mod test {
     #[test]
     #[cfg(unix)]
     fn file() {
-        let surface = SvgSurface::new(100., 100., "/dev/null").unwrap();
+        let surface = SvgSurface::new(100., 100., Some("/dev/null")).unwrap();
```

Comment:
Seems this test need be just `None`, and `Some` need use `tempfile`

Reply:
Yeah, this is probably the reason why it fails.